src/main/resources/db/migration/V0.0.2_002__create_assessment_quest.sql

```diff
@@ -0,0 +1,86 @@
+   create table assessment_questionnaire (
+       id int8 not null,
+        createTime timestamp,
+        createUser varchar(255),
+        deleted boolean not null default false,
+        updateTime timestamp,
+        updateUser varchar(255),
+        language_code varchar(255) not null,
+        name varchar(255) not null,
+        assessment_id int8 not null,
+        questionnaire_id int8 not null,
+        primary key (id)
+    );
+    
+    create table assessment_category (
+       id int8 not null,
+        createTime timestamp,
+        createUser varchar(255),
+        deleted boolean not null default false,
+        updateTime timestamp,
+        updateUser varchar(255),
+        name varchar(255),
+        category_order int4,
+        questionnaire_id int8,
```

Comment:
This field is confusing since it makes me think that `assessment_category` has a relation with the table `questionnaire` but in fact, the relationship is with the table `assessment_questionnaire`.

```suggestion
        assessment_questionnaire_id int8,
```

Reply:
OK